src/index.html

```diff
@@ -1,13 +1,391 @@
+                class="explore__photo grid__item-img_mobile">
+            </div>
+            <div class="grid__item">
+              <img src="./images/old_mistress.png"
+                alt="Woman"
+                class="explore__photo grid__item-img_desktop">
+              <img
+                src="./images/Layer 47@1,5x.png"
+                alt="woman"
+                class="explore__photo grid__item-img_mobile">
+            </div>
+          </div>
+          <h3 class="explore__title">A New Look at Old Masters</h3>
+          <h3 class="explore__subtitle">December 12, 2020 - Through Spring 2023</h3>
+
+          <div class="grid grid_type2">
+            <div class="grid__item">
+              <img src="./images/budha.png"
+                alt="Budha"
+                class="explore__photo">
+            </div>
+            <div class="grid__item">
+              <img
+                src="./images/budda_ph.png"
+                alt="Budha"
+                class="explore__photo">
+            </div>
+            <div class="grid__item">
+              <img
+                src="./images/budda_carpet.png"
+                alt="Budha"
+                class="explore__photo">
+            </div>
+          </div>
+          <h3 class="explore__title">
+            Bodhisattvas of Wisdom, Compassion, and Power
+          </h3>
+          <h3 class="explore__subtitle">
+            March 27, 2021 - October 30, 2022
+          </h3>
+
+          <div class="grid grid_type3">
+            <div class="grid__item">
+              <img
+                src="./images/wall_desktop.png"
+                alt="Candles on the wall"
+                class="explore__photo grid__item-img_desktop">
+              <img
+                src="./images/Layer 51@1,5x.png" alt="wall"
+                class="explore__photo grid__item-img_mobile">
+            </div>
+            <div class="grid__item">
+              <img
+                src="./images/figure__desktop.png"
+                alt="Figurines"
+                class="explore__photo grid__item-img_desktop">
+              <img
+                src="./images/Layer 49@1,5x.png"
+                alt="figurines"
+                class="explore__photo grid__item-img_mobile">
+            </div>
+          </div>
+          <h3 class="explore__title">Inspiring Walt Disney: The Animation of French Decorative Arts</h3>
+          <h3 class="explore__subtitle">December 10, 2021 - March 6, 2022</h3>
+
+          <div>
+            <div class="buttons">
+              <a href="#" class="button">
+                View all exhibitions
+              </a>
+            </div>
+          </div>
+
+        </div>
+      </section>
+
+      <section class="page__section" id="explore">
+        <div class="container">
+          <h2 class="page__section-title">
+            More to Explore Online
+          </h2>
+          <div class="grid grid_type4">
+            <div class="grid__item">
```

Comment:
```suggestion
            <div class="grid__item grid__item--desctop-1-4">
```
в продовження до коментарів по файлу з грідами